hubmapbags/reports.py

```diff
@@ -0,0 +1,110 @@
+from datetime import datetime
+from pathlib import Path
+
+import pandas as pd
+from pandarallel import pandarallel
+from tqdm import tqdm
+
+from . import apis, utilities
+
+
+def __is_protected(hubmap_id, token=None):
+    metadata = apis.get_dataset_info(hubmap_id, instance="prod", token=token)
+    try:
+        return metadata["contains_human_genetic_sequences"]
+    except:
+        return None
+
+
+def __get_published_timestamp(hubmap_id, token=None):
+    try:
+        metadata = apis.get_dataset_info(
+            hubmap_id, instance="prod", token=token, overwrite=False
+        )
+        return datetime.fromtimestamp(metadata["published_timestamp"] / 1000.0)
+    except:
+        return None
+
+
+def __get_created_timestamp(hubmap_id, token=None):
```

Comment:
Is it a problem if a dataset doesn't have certain info, e.g. a created timestamp or type? I can imagine not wanting to halt the program but is it worth printing/logging an error?

Reply:
@gesinaphillips it will return `None`. The purpose of this report is to show missing values too.